src/wallet/Wallet.cpp

```diff
@@ -228,6 +228,32 @@ libmw::WalletBalance Wallet::GetBalance() const
     return balance;
 }
 
+std::vector<libmw::Coin> Wallet::AvailableCoins() const
+{
+    std::vector<libmw::Coin> coins = m_pWalletInterface->ListCoins();
+    std::vector<libmw::Coin> result;
+
+    for (const libmw::Coin& coin : coins) {
+        if (coin.spent_block.has_value()) {
```

Comment:
nit: You may as well just add the `coin.spent` check here instead of at the end, since it doesn't rely on num_confirmations.